etc/systemd/system/50-zfs.preset.in

```diff
@@ -4,6 +4,8 @@ disable zfs-import-scan.service
 enable zfs-import.target
 enable zfs-mount.service
 enable zfs-share.service
+disable zfs-volumes.target
+disable zfs-volume@.service
```

Comment:
not sure if that is needed since the service only comes into effect when some zfs-volume@volname is enabled

Reply:
I will gladly enable this if this is the consensus. We seem to be moving forward very cautiously with this zfs-mount-generator project, and I don't want to be too aggressive.

Reply:
I think it would be reasonable to enable this and the zfs-mount-generator by default in the next tagged release.  Why don't we go ahead and merge this so it's currently disabled, then open a new PR to enable the generators by default.  This way it'll be enabled in the master branch by default well before anything is tagged.

Reply:
I‘m ok with the target disabled but disable the template is nonsense as it does not disable anything ;-)

Reply:
so, line 8 can be dropped but 7 must remain

Reply:
Ah, one more thing... zfs-volume@.service isn't even an installable unit, so there is not need for the disabled line because it can't be enabled anyways

Reply:
Yeah, `disable zfs-volume@*.service` might parse (the documentation on these preset files is pretty sparse), but it's also unnecessary, since it would only exist if you were trying to use this.

Also, line 7 should also go. I added that disable out of an abundance of caution (not creating and enabling a feature). This patch doesn't create that `.target' anymore, so I'm not going to disable it.